packages/react-data-grid-addons/src/editors/ContainerEditorWrapper.js

```diff
@@ -7,16 +7,20 @@ import React, { Component } from 'react';
 // to the required methods.
 module.exports = (ContainerEditor) => {
   return class ContainerEditorWrapper extends Component {
-    getInputNode() {
+    getInputNode = () => {
```

Comment:
We do not need this change, `getValue` and `getInputNode` methods are always used like `this.ref.getValue` so _this_ value will be correct.